samples/serializers/newtonsoft-bson/Newtonsoft_3/Sample/Sample.csproj

```diff
@@ -5,10 +5,9 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.CSharp" />
```

Comment:
This was the only hitch on this one. This reference causes a compilation error but the sample works without it.